services/grpc/grpc.js

```diff
@@ -112,47 +98,62 @@ class ZapGrpc extends EventEmitter {
    * Subscribe to all gRPC streams.
    */
   subscribeAll() {
-    const { Lightning } = this.services
-    this.subscriptions['invoices'] = Lightning.subscribeInvoices()
-    this.subscriptions['transactions'] = Lightning.subscribeTransactions()
-    this.subscriptions['getinfo'] = Lightning.subscribeGetInfo()
-    this.subscribe()
+    this.subscribe('invoices', 'transactions', 'info')
 
     // subscribe to graph updates only after sync is complete
     // this is needed because LND chanRouter waits for chain sync
-    // to complete before accepting subscriptions
+    // to complete before accepting subscriptions.
     this.on('subscribeGetInfo.data', data => {
       const { synced_to_chain } = data
-      if (synced_to_chain && !this.subscriptions['channelGraph']) {
-        const { Lightning } = this.services
-        this.subscriptions['channelGraph'] = Lightning.subscribeChannelGraph()
-        this.subscribe('channelGraph')
+      if (synced_to_chain && !this.subscriptions['channelgraph']) {
```

Comment:
let's use `this.subscriptions.channelgraph` ?

Reply:
Updated